clickhouse-jdbc/src/test/java/com/clickhouse/jdbc/PreparedStatementTest.java

```diff
@@ -0,0 +1,212 @@
+package com.clickhouse.jdbc;
+
+import org.testng.annotations.Test;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.Types;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertNull;
+import static org.testng.Assert.assertTrue;
+
+public class PreparedStatementTest extends JdbcIntegrationTest {
```

Comment:
I would squash all these tests into one because each test will spin up a container just for few queries.

Reply:
actually it will not - one instance of docker env will be reused in all methods in the class.